packages-web/fieldset/src/Fieldset.editorPreview.tsx

```diff
@@ -0,0 +1,32 @@
+import { createElement, ReactElement, ComponentType } from "react";
+
+import { Fieldset } from "./components/Fieldset";
+import { FieldsetPreviewProps } from "../typings/FieldsetProps";
+
+declare function require(name: string): string;
+
+// Temporary interface to fix incorrect widgets property typing for preview ("renderer: ReactNode" should be "renderer: ComponentType")
+interface PreviewProps extends FieldsetPreviewProps {
+    content: {
+        widgetCount: number;
+        renderer: ComponentType;
+    };
+}
+
+export const preview = (props: PreviewProps): ReactElement => {
+    const { class: className, styleObject, legend, content } = props;
```

Comment:
`styleObject` is not available anymore. You need to wait new PR from Mitya (RN 61) and then parse `styles` using `parseStyle` from `@widgets-resources/piw-utils`

Reply:
Using `style` prop and `parseStyle` function now.